demos/asp-net-core/postman-collection.json

```diff
@@ -0,0 +1,66 @@
+{
+	"variables": [],
+	"info": {
+		"name": "ASP.NET Core demo - CompositionGateway",
+		"_postman_id": "71510306-c9e1-ad86-ca26-d98bd6fbfad2",
+		"description": "",
+		"schema": "https://schema.getpostman.com/json/collection/v2.0.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Order #1, SalesOnly",
```

Comment:
Is this designed to be a human friendly name? If so, should it be `"Order #1, Sales only"`?

Reply:
Good point I'll fix it